src/mono/wasm/debugger/BrowserDebugProxy/MonoSDBHelper.cs

```diff
@@ -1821,7 +1835,27 @@ public async Task<JObject> CreateJObjectForValueType(MonoBinaryReader retDebugge
             for (int i = 0; i < numFields ; i++)
```

Comment:
EWorse, even three times! Next line has it as well. I don't see a reason why I used it instead of simply fields[i] - the effect is the same but it's O(1) instead of O(n). Changing.